library/build.gradle

```diff
@@ -10,8 +10,8 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 28
-        versionCode 14
-        versionName "5.0.0"
+        versionCode 15
+        versionName "5.0.1"
```

Comment:
Since you're doing breaking changes, this is probably going to be a major release or a minor release (not a patch bump)?

Reply:
Well, there is not exactly any "breaking changes" as it is the same code with the same Android X libs version, only increased `versionCode` and `versionName`
So I think it is not exactly a "major release" as many changes are just a refactor

Reply:
My point was related to the PR. You're removing this method:

`public void setSkipTextTypeface(@Nullable final String typeURL)`

From the `AppIntro` class right? So whoever is currently using this method on `5.0.0` and will bump to `5.0.1` will not be able to build anymore.

Reply:
I didn't realized that
Sorry, will recover method again and also included new options to change `skipTextTypeface` by using `@FontRes`

Reply:
@cortinico I noticed that I did not remove that method, just change its param from `String` to `Object`, so the user is able to also provide a `@FontRes` 
But perfectly it is possible to define two overloaded methods, where one provides an `String`, other an `Integer`

Reply:
@cortinico methods recovered on commit 83c2ff295ab46bce137b8fc5916788b3597a9447